GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, lead, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision making process. This document is intended to help you understand how

```

Comment:
nit: hyphenate "decision-making" (maybe just my personal preference, but i noticed that wikipedia does it)